src/entities/Bid/types.ts

```diff
@@ -104,6 +105,7 @@ export const BidRequestGeneralInfoSchema = {
       additionalProperties: false,
       required: [...Object.keys(MilestoneItemSchema)],
       properties: MilestoneItemSchema,
+      maxItems: MILESTONE_SUBMIT_LIMIT,
```

Comment:
This is the max they can submit, but afterwards they could create more? Maybe we should also check for the limit in the addMilestone of the project service, after `validateCanEditProject`, something like `validateCanAddMilestone` which checks

`return MILESTONE_SUBMIT_LIMIT > ProjectMilestones.count({project_id})`

Reply:
We don't have a product decision regarding milestones in projects (not proposals).

Reply:
This is a limitation to also include milestones in the snapshot body.